crates/ark/src/lsp/symbols.rs

```diff
@@ -156,6 +156,10 @@ fn collect_symbols(
             collect_sections(node, contents, current_level, symbols)?;
```

Comment:
Definitely agree with our plans to remove local variables from outline, way too noisy

<img width="891" height="848" alt="Image" src="https://github.com/user-attachments/assets/bd0f56c2-f948-4a40-b2d9-a66050215012" />